### PR TITLE
display hashrate per mining thread

### PR DESCRIPTION
**Pull request by barrystyle**

### CODE REVIEW

File: src/core/crypto.cpp [hunk 1]
```diff
@@ -222,6 +223,11 @@ bool verifyHash(SHA256Hash& target, SHA256Hash& nonce, uint8_t challengeSize, bo
 
 
 SHA256Hash mineHash(SHA256Hash target, unsigned char challengeSize, bool usePufferFish) {
+
+    int thrid = 0;
```
Comment (mr-pandabear) [inline src/core/crypto.cpp:227]: rename to "thread_id"

File: src/core/crypto.cpp [hunk 2]
```diff
@@ -234,6 +240,14 @@ SHA256Hash mineHash(SHA256Hash target, unsigned char challengeSize, bool usePuff
     for (size_t i = 32; i < 64; ++i) concat[i] = t_rand() % 256;
 
     while(true) {
+
+        if (hashes++ > 1024) {
+            double hps = (double)hashes / ((double)(get_time_millis() - st) / 1000);
+            Logger::logStatus("[thread" + std::to_string(thrid) + "] mining at " + std::to_string(hps) + "h/sec");
+            hashes = 0;
+            st = get_time_millis();
```
Comment (mr-pandabear) [inline src/core/crypto.cpp:248]: Please use camel case convention "getTimeMilliseconds()"

**Review by mr-pandabear — CHANGES_REQUESTED** [review]

File: src/core/crypto.cpp (hunk 1 above)
Comment (barrystyle) [reply src/core/crypto.cpp]: Resolved in  ebfa87e.

File: src/core/crypto.cpp (hunk 2 above)
Comment (barrystyle) [reply src/core/crypto.cpp]: Resolved in  ebfa87e.

**Comment by mr-pandabear** [comment]
LGTM

**Review by mr-pandabear — APPROVED** [review]

**Pull request merged**